tools/export_wiki_bundle.py

```
@@ -0,0 +1,130 @@
+    repo_root = Path(__file__).resolve().parent.parent
+    wiki_path = resolve_wiki_path(repo_root)
+
+    if wiki_path is None:
+        print("Erro: wiki.md (raiz) ou docs/wiki.md nao encontrado no repo.", file=sys.stderr)
+        return 1
+
+    wiki_text = wiki_path.read_text(encoding="utf-8")
+    raw_paths = parse_paths(wiki_text)
+
+    # Sempre inclui a wiki encontrada e o AGENTS global.
+    paths = [wiki_path.relative_to(repo_root).as_posix(), "AGENTS.global.md"] + raw_paths
```

Ruff (0.15.7)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)